Gruntfile.js

```diff
@@ -3,6 +3,11 @@ module.exports = function(grunt) {
   // The target directory for the final build product.
   var targetDirectory = 'out';
 
+  // Closure dep paths
+  var closureLibraryPath = ['node_modules', 'google-closure-library'].join('/')
```

Comment:
Is 'node_modules' guaranteed to be consistent across node versions?  Or is there an environment variable that is better used?